dmutils/content_loader.py

```diff
@@ -74,6 +103,115 @@ def _question_should_be_shown(self, dependencies, service_data):
                 return False
         return True
 
+    def get_question(self, question_id):
+        for section in self.sections:
+            question = section.get_question(question_id)
+            if question:
+                return question
+
+
+class ContentSection(object):
+    @classmethod
+    def create(cls, section):
+        if isinstance(section, ContentSection):
+            return section
```

Comment:
That's explicitly handled by `_get_section_filtered_by` [here](https://github.com/alphagov/digitalmarketplace-utils/pull/126/files#diff-346b98f8c34fcc00d15a2cc1629cfb6aL52)